### PR TITLE
gegl 0.4.22

### PR DESCRIPTION
**Pull request by ryan-robeson**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by ryan-robeson** [comment]
Upstream switched to Meson. This PR includes a temporary fix for compiling on macOS. See #45841 and https://gitlab.gnome.org/GNOME/gegl/-/issues/214.

**Review by chenrui333 — APPROVED** [review]
Nice!

File: Formula/gegl.rb [hunk 1]
```diff
@@ -32,16 +30,30 @@ class Gegl < Formula
...
-                          "--disable-dependency-tracking",
-                          "--prefix=#{prefix}",
-                          "--disable-docs",
-                          "--without-cairo",
-                          "--without-jasper",
-                          "--without-umfpack",
-                          "--without-libspiro"
-    system "make", "install"
+    args = %W[
+      --prefix=#{prefix}
```
Comment (Bo98) [inline Formula/gegl.rb:34]:
```suggestion
    args = std_meson_args + %W[
```

Comment (ryan-robeson) [reply Formula/gegl.rb]: Got it. Thanks!

**Pull request closed without merging**